### DISCUSSION TITLE
Development version requires newer version of weewx

### DISCUSSION THREAD

**Discussion by boochman** [poblabs/weewx-belchertown#858] [General]
For anyone wanting to run the development release: the README notes that weewx 3.9 or newer is required for the latest release but doesn't note any difference for the development version. I'm using weewx 4.40 and was seeing the following error in my logs along with some (but not all) chart failures:
```
 Traceback (most recent call last):
  File "/Users/Shared/weewx/bin/weewx/reportengine.py", line 196, in run
    obj.start()
  File "/Users/Shared/weewx/bin/weewx/reportengine.py", line 281, in start
    self.run()
 File "/Users/Shared/weewx/bin/user/belchertown.py", line 2609, in run
    aggregate_interval = weeutil.weeutil.nominal_spans(
AttributeError: module 'weeutil.weeutil' has no attribute 'nominal_spans'
```
I copied a newer weeutil.py to my system to fix the issue. It'd probably be a good idea to do a general update, but this was a faster fix. Hopefully I didn't break anything else.

**Comment by poblabs** [comment]
you're right, it's been a while since i had an official release. i should fix this issue, and snapshot what's done thus far as a release. maybe i can try and fit that in the next few days

**Comment by poblabs** [comment]
Oh, also, I just now saw you were trying the development release. You meant the master branch right?

**Reply by boochman** [reply]
Exactly. I used the master zip file link in the README. Everything’s worked great except that one thing, figured I’d let folks know. Even a note I the README might save someone some time, but from my experience, it does look ready for a new release. Thanks for all your work!
